### PR TITLE
Misc cleanups

### PR DESCRIPTION
**Pull request by ubitux**

### CODE REVIEW

File: pynopegl/_pynopegl.pyx [hunk 1]
```diff
@@ -571,7 +571,9 @@ cdef class Scene:
         return _ret_pystr(ngl_scene_dot(self.ctx))
 
     def update_filepath(self, size_t index, const char *filepath):
-        return ngl_scene_update_filepath(self.ctx, index, filepath)
+        cdef int ret = ngl_scene_update_filepath(self.ctx, index, filepath)
+        if ret < 0:
+            raise Exception("unable to update filepath")
```
Comment (mbouron) [inline pynopegl/_pynopegl.pyx:576]: can you add the index and filepath to the exception ?

File: pynopegl/init_header.py [hunk 2]
```diff
@@ -233,11 +233,11 @@ def from_params(
         duration: Optional[float] = None,
         framerate: Optional[Tuple[int, int]] = None,
         aspect_ratio: Optional[Tuple[int, int]] = None,
-    ):
+    ) -> "Scene":
```
Comment (mbouron) [inline pynopegl/init_header.py:236]: pynopegl: add typing information *to* the Scene object ?

File: pynopegl/_pynopegl.pyx (hunk 1 above)
Comment (ubitux) [reply pynopegl/_pynopegl.pyx]: done

File: pynopegl/init_header.py (hunk 2 above)
Comment (ubitux) [reply pynopegl/init_header.py:236]: done

**Review by mbouron — APPROVED** [review]

**Pull request merged**